npm/svelte/rollup.config.mjs

```diff
@@ -1,4 +1,10 @@
 import { createEntries } from '@cypress/mount-utils/create-rollup-entry.mjs'
 
+const config = {
+  external: [
```

Comment:
we did not need svelte as an imported dependency before, but now we need the `mount` and `unmount` functions in svelte 5. Since we want to use whatever version of svelte the user has installed, we want to bundle this as an external dependency